app/bundles/FormBundle/Tests/Controller/Api/FormApiControllerFunctionalTest.php

```diff
@@ -126,29 +115,154 @@ public function testFormWorkflow(): void
+                            'alias'        => 'phone',
+                            'leadField'    => 'companyphone',
+                        ],
+                        [
+                            'label'        => 'Country',
+                            'type'         => 'country',
+                            'alias'        => 'country',
+                            'mappedObject' => 'contact',
+                            'mappedField'  => 'country',
+                        ],
+                    ],
+                    'postAction'  => 'return',
+                ],
+                [
+                    'newName'      => 'Form API test',
+                    'fields'       => [
+                        [
+                            'mappedObject' => 'contact',
+                            'leadField'    => 'email',
+                            'mappedField'  => 'email',
+                        ],
+                        [
+                            'mappedObject' => 'company',
+                            'leadField'    => 'companyaddress1',
+                            'mappedField'  => 'companyaddress1',
+                        ],
+                        [
+                            'mappedObject' => 'company',
+                            'leadField'    => 'companyphone',
+                            'mappedField'  => 'companyphone',
+                        ], [
+                            'mappedObject' => 'contact',
+                            'leadField'    => 'country',
+                            'mappedField'  => 'country',
+                        ],
+                    ],
+                ],
+            ],
+            [
+                [
+                    'name'        => 'Form',
+                    'formType'    => 'standalone',
+                    'isPublished' => true,
+                    'description' => 'Functional API test2',
+                    'fields'      => [
+                        [
+                            'label'        => 'Lastname',
+                            'alias'        => 'lastname',
+                            'type'         => 'text',
+                            'mappedField'  => 'lastname',
+                            'mappedObject' => 'contact',
+                        ],
+                        [
+                            'label'        => 'Company Address',
+                            'type'         => 'text',
+                            'alias'        => 'companyaddress1',
+                            'mappedField'  => 'companyemail',
+                            'mappedObject' => 'company',
+                            'leadField'    => 'companyemail',
+                        ],
+                        [
+                            'label'        => 'Company Phone',
+                            'type'         => 'tel',
+                            'alias'        => 'phone',
+                            'leadField'    => 'position',
```

Comment:
Labels and aliases are confusing here, let's change it to match the mapped field

Reply:
that's right, already corrected